stream_alert_cli/runner.py

```diff
@@ -84,6 +88,25 @@ def lambda_handler(options):
         stream_alert_test(options)
 
 
+def configure_handler(options):
+    """Configure StreamAlert main settings
+
+    Args:
+        options [named_tuple]: ArgParse command result
+    """
+    if options.config_key == 'prefix':
+        if type(options.config_key) not in (unicode, str):
+            LOGGER_CLI.error('Invalid prefix type, must be string')
+            return
+        CONFIG.set_prefix(options.config_value)
+
+    elif options.config_key == 'aws_account_id':
+        if not re.search(r'\A\d{12}\Z', options.config_value):
```

Comment:
@jacknagz one call isn't going to be enough to clock anything 🤣 . Use `timeit`

```
$ python -m timeit --setup="import re; account_id='123456789123'" "re.search(r'\A\d{12}\Z', account_id)" 
1000000 loops, best of 3: 1.03 usec per loop
$ python -m timeit --setup="account_id='123456789123'" "len(account_id) == 12 and account_id.isdigit()"
10000000 loops, best of 3: 0.111 usec per loop
```
1.03 usec w/ `re` and 0.111 usec without